src/style.css

```diff
@@ -1 +1,43 @@
 /* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, sans-serif;
+  font-weight: 300;
+}
+
+body {
+  padding: 20px;
+}
+
+/* container */
+.search-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+/* bar style */
+.search-bar {
+  width: 100%;
+  padding: 10px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  font-size: 16px;
+  transition: borden-color 0.3s ease;
```

Comment:
There is a typo in the 'transition' property. It should be 'border-color' instead of 'borden-color'.